src/redux/slices/bookmarkSlice.js

```diff
@@ -0,0 +1,135 @@
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import axios from "axios";
+import { toast } from "react-toastify";
+
+// export const moveToBookmark = createAsyncThunk("")
+// export const fetchBookmark = createAsyncThunk("post/fetchBookmark", async ({token}) => {
+//     const res = await axios.get("/api/users/bookmark", {
+//         headers: {
+//           authorization: token,
+//         },
+//       });
+//       console.log(res)
+//       console.log("working")
+//     return res;
+//   });
+export const fetchBookmark = createAsyncThunk(
+  "bookmark/fetch",
+  async (data, thunkAPI) => {
+    try {
+      const { token } = data;
+      const res = await axios.get("/api/users/bookmark", {
+        headers: {
+          authorization: token,
+        },
+      });
+      console.log(res);
```

Comment:
please remove

Reply:
I will also remove it
thanks for your goods suggestion